modules/layers/src/column-layer/column-layer.js

```diff
@@ -148,24 +182,63 @@ export default class ColumnLayer extends Layer {
         i++;
       }
     }
-    model.setProps({geometry});
+
+    const geometries = Object.keys(drawLines).map(mode => {
```

Comment:
It is not going to have a lot of modes, since we only have `stroke` and `fill` right now, it is better to explicitly `stroke*` and `fill*`.